src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -1,20 +1,346 @@
 # New Tracker
 
-  * Describe /tracker as a group of new tracker endpoints, where there are some new changes
-  * List changes we have made between 2.35->2.36
-  * Make a note that the old endpoints are marked as deprecated, but still work. Not all the functionality is ready in the new endpoint yet.
-
-## Tracker Import
+In version V2.36 DHIS2 introduced a set of brand new tracker endpoints dedicated to importing and querying tracker entities (tracked entity, enrollments, events, relationships, …).
```

Comment:
Do not need to put the dots at the end, there are only 4 tracker entities

Reply:
fair enough :)